ansible_collections/arista/avd/roles/eos_designs/python_modules/core_interfaces_and_l3_edge/ethernet_interfaces.py

```diff
@@ -29,6 +29,9 @@ def ethernet_interfaces(self) -> list | None:
                 ethernet_interface = self._get_common_interface_cfg(p2p_link)
                 ethernet_interface.update(self._get_ethernet_cfg(p2p_link))
 
+                if self.shared_utils.underlay_isis is True:
+                    ethernet_interface.update({"isis_enable": self.shared_utils.isis_instance_name, "isis_bfd": get(self._hostvars, "underlay_isis_bfd")})
```

Comment:
This enabled isis unconditionally. Please remove this and find the place where we enable isis today, and add the bfd there.

Reply:
Fixed!